docs/contributing.rst

```diff
@@ -204,7 +204,7 @@ A checklist for creating, testing, and distributing a new version.
 #. Add a ``:release:`` line to :file:`docs/changelog.rst`.
 #. Commit and open a pull request for review.
 #. Merge the pull request, and ensure the `Travis`_ build passes.
-#. Create a new git tag with ``git tag -m tag {version}``.
+#. Create a new git tag with ``git tag -m "Release v{version}" {version}``.
```

Comment:
This should be `-sam` to sign the tag, and have it be an annotated tag rather than a lightweight tag. (This affects how tags display in GitHub.)

Reply:
(And also means the tag gets a date on when it was created, lightweight tags don't get one)

Reply:
`-a` is redundant to both `-m` and `-s`.

I removed `"Release v{version}"` to avoid redundant information and extra unnecessary typing. I removed `-s` because it implies a whole workflow around GPG signing which drastically limits the workflows ([more details here](https://github.com/jaraco/keyring/issues/427)).  I'm okay with recommending signing tags, but if you require it, it adds a lot of implied steps to the process.

Reply:
Verified that`-m` implies `-a`: https://git-scm.com/docs/git-tag

Re: signing and message, I reviewed other PyPA projects. It looks like most of them do not consistently/frequently sign tags (with [packaging](https://github.com/pypa/packaging/tags) being a notable exception). Messages are more of a mixed bag; some seem to be using lightweight tags that default to the commit message. Some (notably [pip](https://github.com/pypa/pip/tags)) use a `Release {version}` message.

The previous instruction of `git tag -m tag {version}` doesn't make sense to me, because it results in a message of `tag`, which is what prompted the change. For a truly empty message on an annotated tag, I think the instruction would have to be `git tag -m "" {version}`. Personally, I find the the `Release v{version}` slightly friendlier, if redundant.

With all that in mind, I'm inclined to merge this PR as-is, and use its instructions to publish the release.